src/xref2/tools.ml

```diff
@@ -1882,6 +1886,13 @@ and fragmap :
     map_module_decl m.type_ new_subst >>= fun type_ ->
     Ok (Left { m with type_ })
   in
+  let map_removed = function
```

Comment:
Why are we doing this instead of creating the correct item in the first place?

Reply:
answer: because we need to do `sub_of_removed` and that needs the Ident.t rather than just the name.